packages/openapi-to-graphql/test/example_api.test.ts

```diff
@@ -1982,9 +1982,9 @@ test('Option selectQueryOrMutationField', () => {
   }`
 
   // The users field should exist as a Query field
-  const promise = graphql(createdSchema, query).then((result) => {
+  const promise = graphql({ schema: createdSchema, source: query }).then((result) => {
     expect(
-      result.data['__schema'].queryType.fields.find((field) => {
+      result.data['__schema']['queryType'].fields.find((field) => {
```

Comment:
It seems strange to me to change `.queryType` to `['queryType']` but not `.fields` to `['fields']`. I can see that the code was not consistent to begin with. I am in the opinion that the dot notation is better for brevity.

Reply:
This change was needed to resolve `TS2339: Property 'queryType' does not exist on type 'unknown'`.